quarkus-integration/quarkus/deployment/src/main/java/ai/timefold/solver/quarkus/deployment/TimefoldProcessor.java

```diff
@@ -525,15 +529,56 @@ private GeneratedGizmoClasses generateDomainAccessors(SolverConfig solverConfig,
         Set<String> generatedMemberAccessorsClassNameSet = new HashSet<>();
         Set<String> gizmoSolutionClonerClassNameSet = new HashSet<>();
 
+        /*
+         * TODO consistently change the name "entity" to something less confusing
```

Comment:
What exactly does it enhance? Only things that may change during solving (solution, genuine and shadow entities) or even facts?
Trying to figure out what the name should be.